WordPress/Classes/ViewRelated/Post/PostCardTableViewCell.m

```diff
@@ -359,20 +359,15 @@ - (void)configureMetaButtons
     [self resetMetaButton:self.metaButtonRight];
     [self resetMetaButton:self.metaButtonLeft];
 
-    // We don't have comment and like counts for self-hosted sites.
-    if (![self.contentProvider isWPcom]) {
-        return;
-    }
-
     NSMutableArray *mButtons = [NSMutableArray arrayWithObjects:self.metaButtonLeft, self.metaButtonRight, nil];
-    if ([self.contentProvider numberOfComments] > 0) {
+    if ([self.contentProvider supportsComments] && [self.contentProvider numberOfComments] > 0) {
```

Comment:
You could say the same about the previous `isWPcom` check :wink: 
I don't know, they're probably not necessary, but it seemed good to add the explicit check